backend/app/tests/utils/test_db.py

```
@@ -332,3 +422,83 @@
+        from sqlmodel import Session, SQLModel, create_engine
+        from app.models.prompt import Prompt, Tag, PromptTagLink
+
+        # 获取测试数据库引擎
+        test_db_url = get_test_db_url()
+        engine = create_engine(test_db_url)
+        
+        # 创建基础测试数据
+        with Session(engine) as session:
+            # 检查是否已存在测试数据
+            existing_tags = session.exec("SELECT COUNT(*) FROM tag").first()
+            
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)